js/client.js

```diff
@@ -967,7 +967,9 @@ function toId() {
 			} else if (data.substr(0, 3) === '|N|') {
 				var names = data.substr(1).split('|');
 				if (app.ignore[toUserid(names[2])]) {
-					app.ignore[toUserid(names[1])] = 1;
+					app.ignore[toUserid(names[1])] = {
+						notified: false,
```

Comment:
```suggestion
						notified: app.ignore[toUserid(names[2])].notified,
```

Reply:
Yeah, I was thinking about that originally when I wrote that. But I felt it would be better to err on the side of "make sure they're aware of this."

Reply:
Definitely no. It's better to err on the side of "don't annoy them with notifications". The only reason we're notifying at all is so people don't accidentally ignore the wrong person and never realize, or if a friend messes with their settings they still know.

Reply:
Okay.